### PR TITLE
Query update

### PR DESCRIPTION
**Pull request by Daphineatim**
### In this milestone, the following changes were implemented;

- Added a column species of type string to your animals table. Modified your schema.sql file.
-  Inserted more data in data.sql file.
-  Added transactions to the queries.sql file.

query1

query2

query3

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -1,11 +1,13 @@
...
 CREATE TABLE animals
 (
 	id int GENERATED ALWAYS AS IDENTITY,
     name varchar,
     date_of_birth date,
     escape_attempts int,
     neutered boolean,
-    weight_kg decimal
+    weight_kg decimal,
+    species varchar
```
Comment (Reem-lab) [inline schema.sql]: - [ ] I think this is not the right way to add a colmun you should use the alter command so you can add a column to an existed table so kindly use the alter command for adding a new column. ✅🙏🏻

You can add somthing like this
> ALTER TABLE animals ADD species varchar(255);

**Review by Reem-lab — CHANGES_REQUESTED** [review]
# Hi @Daphineatim, 👋🏻👋🏻 

Good job so far! ✨💯🔥🔥
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
- No Linters error ✅
- Followed GitFlow ✅
- Great working on the functionality ✅
- Descriptive PR ✅

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: schema.sql (hunk 1 above)
Comment (Daphineatim) [reply schema.sql]: I have fixed the issue, thank you so much for the feedback.

**Review by prolajumokeoni — APPROVED** [review]
Hi @Daphineatim 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
## Highlights 
- Git flow was used correctly 🚀 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**